### DISCUSSION TITLE
About i90 files

### DISCUSSION THREAD

**Discussion by wiring1881** [krvajal/vscode-fortran-support#1102] [Q&A - deprecated see Fortran Discourse] [answered]
Is there a way to enable syntax highlighting in vscode-fortran-support for i90 files (intermediate files generated from ifx/ifort compiler)? and in addition even formatting for that? In other words, treat these files just as free-form f90 files.

**Comment by gnikit** [comment, marked as answer]
This should work for both highlighting and formatting.

```json
  "files.associations": {
    "*.i90": "FortranFreeForm"
  },
```
